CSharp/DevVmPowershell/Helpers/Implementations/InstanceSettingsHelper.cs

```diff
@@ -4,35 +4,61 @@
 using Relativity.Services.ServiceProxy;
 using System;
 using System.Linq;
+using System.Net.Http;
+using Helpers.RequestModels;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Helpers.Implementations
 {
 	public class InstanceSettingsHelper : IInstanceSettingsHelper
 	{
 		private ServiceFactory ServiceFactory { get; }
+		private string InstanceAddress { get; set; }
+		private string AdminUsername { get; set; }
+		private string AdminPassword { get; set; }
 
-		public InstanceSettingsHelper(IConnectionHelper connectionHelper)
+		public InstanceSettingsHelper(IConnectionHelper connectionHelper, string instanceAddress, string adminUsername, string adminPassword)
 		{
 			ServiceFactory = connectionHelper.GetServiceFactory();
+			InstanceAddress = instanceAddress;
+			AdminUsername = adminUsername;
+			AdminPassword = adminPassword;
 		}
 
 		public int CreateInstanceSetting(string name, string section, string description, string value)
 		{
 			try
 			{
-				using (IInstanceSettingManager instanceSettingManager = ServiceFactory.CreateProxy<IInstanceSettingManager>())
+				HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+				string createUrl = $"Relativity.REST/api/Relativity.InstanceSettings/workspace/{Constants.EDDS_WORKSPACE_ARTIFACT_ID}/instancesettings/";
```

Comment:
I would move all the URL's to the Constants class so they are in one place for easy updates in future.

- [ ] Move URL to Constants class